### ISSUE TITLE
NTR: 7-hydroxyoctanoic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8598]
New term request 7-hydroxyoctanoic acid high in urine (segal_030513144052):

**Comment by MickeySegal** [comment]
https://foodb.ca/compounds/FDB022069 lists 7-Hydroxyoctanoic as "linked to the inborn metabolic disorder medium chain acyl-coa dehydrogenase deficiency" but doesn't go into further details.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000605
- label: Elevated urinary 7-hydroxyoctanoic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: 
- def: The amount of 7-hydroxyoctanoic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8598
- synonyms:
